ml/src/test/scala/ai/h2o/sparkling/ml/algos/BinomialPredictionTestSuite.scala

```diff
@@ -87,10 +89,10 @@ class BinomialPredictionTestSuite extends FunSuite with Matchers with SharedH2OT
 
     val datasetFields = dataset.schema.fields
     val labelField = StructField("label", StringType, nullable = false)
-    val propFields = Seq("p0", "p1").map(StructField(_, DoubleType, nullable = false))
+    val probabilitiesField = StructField("probabilities", MapType(StringType, DoubleType, valueContainsNull = false), nullable = false)
```

Comment:
`MapType` uses a linear key search. Do we know how many classes we can have in real world?

Reply:
Good to know! I don't think that in real life use-cases there are that many classes. Plus this is still part of the detailed prediction column whilst people who care about performance will probably still use just the regular `prediction` column